.github/workflows/Procedure_push_docker_images.yml

```diff
@@ -1,33 +1,84 @@
-run-name: Launch Push Docker Images - Executed by @${{ github.actor }}
+run-name: Launch Push Docker Images - ${{ inputs.id }}
 name: Push Docker Images
 
 on:
   workflow_dispatch:
     inputs:
-      IMAGE_TAG:
+      image_tag:
         description: 'Docker image tag'
         default: '4.10.0'
         required: true
-      DOCKER_REFERENCE:
+      docker_reference:
         description: 'wazuh-docker reference'
         default: 'v4.10.0'
+        required: false
+      products:
+        description: 'Comma-separated list of the image names to build and push'
+        default: 'wazuh-manager,wazuh-dashboard,wazuh-indexer'
+        required: true
+      filebeat_module_version:
+        description: 'Filebeat module version'
+        default: '0.4'
+        required: true
+      revision:
+        description: 'Package revision'
+        default: '1'
+        required: true
+      push_images:
+        description: 'Push images'
+        type: boolean
+        default: true
         required: true
-      PRODUCTS:
+      id:
+        description: "ID used to identify the workflow uniquely."
+        type: string
+        required: false
+      dev:
+        description: "Add tag suffix '-dev' to the image tag ?"
+        type: boolean
+        default: false
```

Comment:
Change the `dev` default value to true for the workflow_dispatch.

Reply:
Done ✅